cmd/minikube/cmd/start.go

```diff
@@ -141,6 +145,15 @@ func runStart(cmd *cobra.Command, args []string) {
 		glog.Errorln("Error starting the VM: ", err)
 		os.Exit(1)
 	}
+
+	// Set Proxy to as Shell Env
+	if viper.IsSet("http-proxy") || viper.IsSet("https-proxy") {
```

Comment:
Wouldn't it make sense to move this into cluster.StartHost? In fact, we might want to add the proxy options to _cluster.MachineConfig_? WDYT?